test/integration_test/migration_test.go

```diff
@@ -456,6 +457,19 @@ func migrationLabelExcludeSelectorTest(t *testing.T) {
 	)
 }
 
+func namespaceLabelSelectorTest(t *testing.T) {
+	triggerMigrationTest(
+		t,
+		"ns-selector-test",
+		"cassandra",
+		[]string{"mysql-1-pvc"},
+		"namespace-selector-migration",
+		true,
+		true,
+		true,
+	)
```

Comment:
Lets add another app in a different namespace and ensure that that app is not migrated and cassandra gets migrated

Reply:
I have modified the test to run two apps in different namespace and only migrate cassandra for which namespace label matches.